CONTRIBUTING.md

```diff
@@ -12,7 +12,7 @@ Want to contribute by creating a pull request? Great! [Fork iTwinUI-react](https
 
 ## How to setup
 
-To clone and build iTwinUI-react, you'll need [Git](https://git-scm.com) and [Yarn 1](https://yarnpkg.com/getting-started/install) installed on your computer.
+To clone and build iTwinUI-react, you'll need [Git](https://git-scm.com), [Yarn 1](https://yarnpkg.com/getting-started/install) and [Node 16](https://nodejs.org/en/download/current/) installed on your computer.
```

Comment:
Unrelated but I noticed the yarn 1 link is pointing to the wrong place. Can you update it to https://classic.yarnpkg.com/en/docs/install?

Also I think node should be mentioned before yarn because it is a prereq

Reply:
Ohh, good points, thanks. I changed the yarn link and put node before yarn.

I also noticed that the old node link was for the current version (18+). So I replaced the old link (https://nodejs.org/en/download/current/) with the link to the LTS version (16.17.1) (https://nodejs.org/en/download/). Is this alright?

Reply:
Looks good